src/core/string.h

```diff
@@ -53,6 +53,6 @@ int string_to_int(const uint8_t *str);
  * @param force_plus_sign Force plus sign in front of positive value
```

Comment:
Since we explain the parameters here, please add a comment for the `thousands_separator`.